std_msgs/Header header
bool all_speech
bool contains_speech
duration speech_duration
duration time_since_speech
